modules/ROOT/pages/design-create-publish-api-visual-editor.adoc

```diff
@@ -3,14 +3,14 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can use API Designer's visual API editor to create RAML or OAS API specifications by populating language-neutral forms.
+You can use API Designer's visual API editor to scaffold RAML or OAS API specifications by populating language-neutral forms.
 
 == Before you begin
 Within Anypoint Platform, your user ID must be assigned the _Design Center Developer_ permission.
 
 == About this task
 
-As you populate the forms, the visual API editor generates two API specifications: one in RAML 1.0, the other in OAS 2.0 (in JSON). Both are visible in a read-only pane on the right of the screen. If at any time you want to edit the specification directly, click *Edit RAML*  or *Edit OAS*to switch to the API editor. After you switch, however, you cannot switch back to the visual API editor to continue creating the API specification.
+As you populate the forms, the visual API editor generates two API specifications: one in RAML 1.0, the other in OAS 2.0 (in JSON). Both are visible in a read-only pane on the right of the screen. If at any time you want to edit the specification directly or customize it (import RAML fragments, add a security scheme, and more), click *Edit RAML*  or *Edit OAS* to switch to the API editor. After you switch, however, you cannot switch back to the visual API editor to continue creating the API specification.
```

Comment:
Again, I would remove the "add security schemes" limitation as, in fact, we will be adding this in the next version

Reply:
Done.